cli/web_worker.rs

```diff
@@ -77,19 +79,18 @@ mod tests {
       state,
     );
     worker.execute("bootstrapWorkerRuntime(\"TEST\")").unwrap();
-    worker.execute("runWorkerMessageLoop()").unwrap();
     worker
   }
-
+  // FIXME(bartlomieju)
+  #[ignore]
   #[test]
```

Comment:
Disabled test which is also modified. Are these changes intentional?

Reply:
Fixed